src/core/llm/README.md

```
@@ -0,0 +1,492 @@
+- **Vercel**: Image support through underlying provider capabilities
+- **Custom Providers**: Implement image handling as needed
+
+### Custom Provider Integration
+
+See `src/core/llm/services/README.md` for provider integration steps.
+
+// 1. Add to registry (in registry.ts)
+// Note: LLM_REGISTRY is const, so this requires modifying the source
+// Add your provider to the LLM_REGISTRY object:
+<!-- provider integration block removed -->
+    models: [
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)